src/staticanalysis/bot/static_analysis_bot/coverity/coverity.py

```diff
@@ -180,7 +180,19 @@ def return_issues(self, coverity_results_path, revision):
                 logger.info('Coverity did not find any issues')
                 return []
 
-            return [CoverityIssue(issue, revision) for issue in result['issues']]
+            return [CoverityIssue(issue, revision) for issue in result['issues'] if self._is_local_issue(issue)]
+
+    def _is_local_issue(self, issue):
+        '''
+        The given coverity issue should be only locally stored and not in the
+        remote snapshot
+        '''
+        stateOnServer = issue['stateOnServer']
+        # According to Coverity manual:
+        # presentInReferenceSnapshot - True if the issue is present in the reference
+        # snapshot specified in the cov-run-desktop command, false if not.
+        return stateOnServer is not None and 'presentInReferenceSnapshot' in stateOnServer and stateOnServer[
```

Comment:
Please use an explicit line return instead of breaking the line:

```python
return stateOnServer is not None and 'presentInReferenceSnapshot' in stateOnServer \
  and stateOnServer['presentInReferenceSnapshot'] is False
```

Reply:
sure check my last commit.